Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py

```diff
@@ -264,7 +258,6 @@ def fetch_indicators_command(client: Client, default_indicator_type: str, **kwar
                         'fields': create_fields_mapping(raw_json, mapping) if mapping else {}
                     }
                     indicators.append(indicator)
-
     return indicators
 
 
```

Comment:
`itype` in `get_indicators_command` should behave the same way as `fetch_indicators_command` and since you've updated that to check `auto_detect_type`, that should also happen here.

Reply:
You basically need to make sure that `get_indicators_command` also takes into account the `auto_detect_type` param the same way you've incorporated it for fetching since the behavior should be aligned across the two functions. That goes for all three modules.